themes/bootstrap3/templates/notifications/pagesform.phtml

```diff
@@ -54,9 +54,9 @@ $form->prepare();
 
           <div class="form-group">
               <?php $content = $form->get('content_' . $language); ?>
-              <?php $content->setLabelOptions(['disable_html_escape' => true]); ?>
               <?= $this->formLabel($content) ?>
               <?= $this->formElement($content) ?>
+              <a href="<?php echo $this->url('help-home')?>?topic=markdown" data-lightbox class="help-link"><?=$this->transEsc('Markdown and emoji help')?></a>
```

Comment:
It might be slightly preferable to use the helper to apply the topic parameter, though I realize in the current codebase this is not consistently done everywhere:

```
<?=$this->url('help-home', [], ['query' => ['topic' => 'markdown']])?>
```